electrolyzer/supervisor.py

```diff
@@ -137,6 +141,21 @@ def update_stack_status(self):
                 self.waiting[i] = 0
                 self.stacks_waiting_vec[i] = 0
 
+    def initialize_plant_stacks(self):
+        # TODO: decide how many stacks should be turned on
+        stack_number = round(self.initial_power_kW / self.stack_rating_kW) + 1
+        if stack_number > self.n_stacks:
+            stack_number = int(self.n_stacks)
```

Comment:
isn't `self.n_stacks` an int already?

Reply:
Yeah, it is, I think I just copy and pasted